src/sources/audiosourceoggvorbis.cpp

```diff
@@ -0,0 +1,160 @@
+#include "sources/audiosourceoggvorbis.h"
+
+namespace Mixxx {
+
+namespace {
+
+const int kLogicalBitstreamIndex = -1; // whole stream/file
```

Comment:
this should be 
kCurrentBitstreamLink 
https://xiph.org/vorbis/doc/vorbisfile/ov_info.html
and 
kEntireBitstreamLink 
https://xiph.org/vorbis/doc/vorbisfile/ov_pcm_total.html